website/docs/docs/deploy/job-scheduler.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "The dbt Cloud Scheduler"
+id: "job-scheduler"
+sidebar_label: "Job scheduler"
+description: "The dbt Cloud Scheduler queues scheduled or API-triggered runs, before preparing the job to enter cloud data platform. Build observability into transformation workflows with the in-app scheduling, logging, and alerting." 
+tags: [scheduler]
+---
+
+The Scheduler is the backbone of running jobs in dbt Cloud, bringing power and simplicity to building data pipelines in both continuous integration and production contexts. The Scheduler frees teams from having to build and maintain their own infrastructure and ensures the timeliness and reliability of data transformations.
+
+The Scheduler enables both cron-based and event-driven execution of dbt commands in the user’s data platform. Specifically, the Scheduler handles:
+
+- Cron-based execution of dbt Cloud jobs that run on a predetermined cadence
+- Event-driven execution of dbt Cloud CI jobs triggered by pull requests to the dbt repo
+- Event-driven execution of dbt Cloud jobs triggered by API
+- Event-driven execution of dbt Cloud jobs manually triggered by a user to "Run Now"
+
+:::tip
+Actions in the dbt Cloud IDE aren't included in the Scheduler's handling. For example, dbt commands executed in the IDE don't affect the Scheduler or your run slots. 
```

Comment:
hey @will-sargent-dbtlabs fyi made this update - let me know what you think!

Reply:
is this a common misconception for people? This tip made me more confused, and would prefer to leave out unless there's more context

Reply:
should've provided more context, sorry. @will-sargent-dbtlabs mentioned he has received questions from users on whether ide runs use run slots.